consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -1417,52 +1413,32 @@ impl<
                 .await
                 .unwrap();
 
-            // If `>= f+1` notarized a given proposal, then we should backfill missing
-            // notarizations
+            // If at least one honest node notarized a proposal in this view,
+            // then backfill any missing certificates.
             let round = self.views.get(&view).expect("missing round");
-            if let Some(parent) = round.at_least_one_honest() {
-                if parent >= self.last_finalized {
```

Comment:
this line always returns true under the assumptions of consensus and honest behavior, so don't check it and just assume you fall into this conditional (and that the `else` below is dead code)